MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
 
 Derive additional information from the data like sum, average, maximum, minimum, and count using summaries in the data grid. These summary values can be computed for groups or for the entire control using `DataGridSummaryRow` and `DataGridSummaryColumn` that implements [ISummaryRow](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ISummaryRow.html) and [ISummaryColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ISummaryColumn.html) interfaces.
 
 N> The Summary does not refresh with data. To update the summary for the newly added row, or for the modified summary column, set the [SfDataGrid.View.LiveDataUpdateMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.LiveDataUpdateMode.html) to `LiveDataUpdateMode.AllowDataShaping` or `LiveDataUpdateMode.AllowSummaryUpdate`.
 
+## Caption summaries
+
+The data grid provides built-in support for caption summaries. The caption summary value calculated based on the records in a group. The summary information will be displayed in the caption of group.
+
+### Formatting built-in caption summary
+
+By default, caption summary rows are displayed with the [SfDataGrid.GroupCaptionTextFormat]() property.
+
+Default group caption format is `{ColumnName}: {Key} - {ItemsCount} Items`.
+
+* **ColumnName**: Displays the name of the currently grouped column.
+* **Key**: Displays the key value of the group.
+* **ItemsCount**: Displays the number of items in a group.
+
+![DataGrid with formatting in caption summary row](Images\Caption-Summary\ShowSummaryInRow.png)
+
+The group caption text format can be customized by setting the `SfDataGrid.GroupCaptionTextFormat` property. The following code example illustrates how to customize group caption text in the data grid:
+
+{% tabs %}
+{% highlight xaml%}
+<sfGrid:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumns="True"
+                   ColumnSizer="Star"
+                   GroupCaptionTextFormat="{}{ColumnName}: {Key}">
+{% endhighlight %}
+
+{% highlight c#%}
+//Customized group caption text
+dataGrid.GroupCaptionTextFormat = "{ColumnName} : {Key}";
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the outcome of the previous code:
+
+![DataGrid with formatting in caption summary row](Images\Caption-Summary\ShowSummaryInRow2.png)
+
+
+### Displaying summary for a row
+
+Display summary information in a row by setting the [DataGridSummaryRow.ShowSummaryInRow]() property to `true` and define summary columns. You have to define the [DataGridSummaryRow.Title]() based on the [DataGridSummaryColumn.Name]() property to format summary columns value in a row.
```

Comment:
To display summary information in a row, you can set the [DataGridSummaryRow.ShowSummaryInRow](https://chat.openai.com/) property to true and define summary columns. Additionally, you need to define the [DataGridSummaryRow.Title](https://chat.openai.com/) based on the [DataGridSummaryColumn.Name](https://chat.openai.com/) property in order to format the values of the summary columns in the row.

Reply:
Modified.